src/schemata/definitions/workflow_template/changeReason.yml

```diff
@@ -1,17 +1,14 @@
 ---
 title: Change Reason
 description: |
-  A flag indicating that overwriting an existing field value requires a reason and prompts the user with a reason dialog. The reason is recorded along with the new field value. If no reason is submitted, the field is not updated.
-
-  Updating a field that has no value with a value does not require a reason. A field update that changes the value to `null`, however, does require a reason. Consider the following example:
```

Comment:
IIUC, this is no longer the truth? I.E. change reasons must be provided every time there is a change after the initial value has been set?

Reply:
> The first update to a field's value never requires a reason, any further updates always do.

Doesn't matter if the field has a `defaultValue` or not to begin with, it's about the first and all subsequent updates

Reply:
While the text description was correct, I just realized that the step-by-step example was incorrect. I pushed the change.